README.md

```diff
@@ -85,15 +85,26 @@ java -DdebugOperations -cp /path/to/openapi-generator-cli.jar:/path/to/your.jar
 Will, for example, output the debug info for operations.
 You can use this info in the `api.mustache` file.
 
+## Local Test Setup for twilio-go
+To run the unit tests or integration tests locally you need complete the following steps:
+1. Copy the generated code in the `codegen/twilio` directory and move it into the `examples/go` package. This should resolve any import errors in the tests files. If you are missing the codegen directory you can skip moving any files and run `make install` then run the command below:
```

Comment:
Doesn't running ./prism.sh do all of this for you?

Reply:
Yup! I wasn't sure if explicitly calling out the steps that `./prism.sh` does for you would help outside developers while testing. I'm happy to change this step to "Run the `.prism.sh` file".

Reply:
Yes the prism.sh does this but it places everything within the codegen directory. If you want to run everything locally you still need to have the generated code in the oai folder in the `examples/go` package.

Reply:
Could you add a step before [this](https://github.com/twilio/twilio-oai-generator/blob/main/prism.sh#L11) line to do `cp -R codegen/ examples/go` and get rid of this line [here](https://github.com/twilio/twilio-oai-generator/blob/main/Dockerfile#L3)? That way we are able to run our tests without doing any manual copies outside docker.